### ISSUE TITLE
Plugin initialisation crashing since 0.11.15 release

### ISSUE THREAD

**Issue by neil-williamson** [volterraedge/terraform-provider-volterra#173]
```
╷
│ Error: Plugin did not respond
│ 
│   with provider["registry.terraform.io/volterraedge/volterra"],
│   on f5stub.tf line 9, in provider "volterra":
│   9: provider "volterra" {
│ 
│ The plugin encountered an error, and failed to respond to the
│ plugin.(*GRPCProvider).ConfigureProvider call. The plugin logs may contain
│ more details.
╵

Stack trace from the terraform-provider-volterra_v0.11.15 plugin:

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xe626b3]

goroutine 179 [running]:
gopkg.volterra.us/stdlib/util/tls.(*CertManager).SetStopCh(0x0, 0xc0001096e0)
	gopkg.volterra.us/stdlib@v0.0.0-20221004231356-fdd845c75d8a/util/tls/cert_manager.go:83 +0x33
gopkg.volterra.us/stdlib/util/tls.(*CertManager).StartExpiryRotater(0x0)
	gopkg.volterra.us/stdlib@v0.0.0-20221004231356-fdd845c75d8a/util/tls/cert_manager.go:174 +0x38
gopkg.volterra.us/stdlib/client.(*Client).ConnectRest(0xc000730510, {0xbf01d18, 0xc000130000})
	gopkg.volterra.us/stdlib@v0.0.0-20221004231356-fdd845c75d8a/client/client.go:413 +0x87
gopkg.volterra.us/stdlib/client.NewClient({0xc00295c000, 0xc0007341c0}, {0xc0003718a8, 0x75060c0, 0xc0000d4801})
	gopkg.volterra.us/stdlib@v0.0.0-20221004231356-fdd845c75d8a/client/client.go:617 +0x756
gopkg.volterra.us/stdlib/client/vesapi.NewAPIClient({0xc00295c000, 0x27}, 0xc000134000, {0xc0007341c0, 0xc00031e300, 0xb2})
	gopkg.volterra.us/stdlib@v0.0.0-20221004231356-fdd845c75d8a/client/vesapi/vesapi.go:382 +0x33a
github.com/volterraedge/terraform-provider-volterra/volterra.(*Config).Client(0xc0003719e0)
	github.com/volterraedge/terraform-provider-volterra/volterra/config.go:237 +0x85
github.com/volterraedge/terraform-provider-volterra/volterra.providerConfigure(0x7df1aac, {0x18, 0x0})
	github.com/volterraedge/terraform-provider-volterra/volterra/provider.go:178 +0x547
github.com/volterraedge/terraform-provider-volterra/volterra.Provider.func1(0xc002234780)
	github.com/volterraedge/terraform-provider-volterra/volterra/provider.go:123 +0x3c
github.com/hashicorp/terraform-plugin-sdk/helper/schema.(*Provider).Configure(0xc00292fe80, 0xc0016e0570)
	github.com/hashicorp/terraform-plugin-sdk@v1.1.1/helper/schema/provider.go:275 +0xb4
github.com/hashicorp/terraform-plugin-sdk/internal/helper/plugin.(*GRPCProviderServer).Configure(0xc00223a048, {0xc00223c240, 0x4bc526}, 0xc00223c240)
	github.com/hashicorp/terraform-plugin-sdk@v1.1.1/internal/helper/plugin/grpc_provider.go:487 +0x1d2
github.com/hashicorp/terraform-plugin-sdk/internal/tfplugin5._Provider_Configure_Handler({0x7ad3880, 0xc00223a048}, {0xbf01d88, 0xc0018bd7d0}, 0xc0000f2840, 0x0)
	github.com/hashicorp/terraform-plugin-sdk@v1.1.1/internal/tfplugin5/tfplugin5.pb.go:3135 +0x170
google.golang.org/grpc.(*Server).processUnaryRPC(0xc00184d180, {0xbf9ed68, 0xc0010cbe00}, 0xc0022366c0, 0xc0025dd770, 0x108a4038, 0x0)
	google.golang.org/grpc@v1.40.0/server.go:1297 +0xccf
google.golang.org/grpc.(*Server).handleStream(0xc00184d180, {0xbf9ed68, 0xc0010cbe00}, 0xc0022366c0, 0x0)
	google.golang.org/grpc@v1.40.0/server.go:1626 +0xa2a
google.golang.org/grpc.(*Server).serveStreams.func1.2()
	google.golang.org/grpc@v1.40.0/server.go:941 +0x98
created by google.golang.org/grpc.(*Server).serveStreams.func1
	google.golang.org/grpc@v1.40.0/server.go:939 +0x294

Error: The terraform-provider-volterra_v0.11.15 plugin crashed!

This is always indicative of a bug within the plugin. It would be immensely
helpful if you could report the crash with the plugin's maintainers so that it
can be fixed. The output above should help diagnose the issue.
```
Content of `f5stub.tf` for reference.  This occurs regardless of whether `enable_f5xc` is `true` or `false`:
```
locals {
  enable_f5xc = tobool(data.google_secret_manager_secret_version.enable_f5xc.secret_data)
}

data google_secret_manager_secret_version enable_f5xc {
  secret = "enable_f5xc"
}

provider "volterra" {
  url = "https://(redacted).console.ves.volterra.io/api"
  # API keys provided by Cloud Build environment variables in appropriate envs
  # To ignore the provider where we're not building it, we use dummy key material to satisfy attribute requirements
  api_key  = local.enable_f5xc ? null : "f5stub_dummy_key.pem"
  api_cert = local.enable_f5xc ? null : "f5stub_dummy_cert.pem"
}
```

**Comment by madhukar32** [comment]
Looking at this

**Comment by madhukar32** [comment]
@neil-williamson: This is fixed in 0.11.16 release
